sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -219,4 +219,11 @@ class CliSuite extends SparkFunSuite with BeforeAndAfter with Logging {
         -> "OK"
     )
   }
+
+  test("SPARK-11188 Analysis error reporting") {
+    runCliWithin(2.minute)(
+      "select * from nonexistent_table;"
+        -> "Error in query: Table not found: nonexistent_table;"
```

Comment:
@marmbrus 
Thanks for your quick feedback. Actually this is one of the question i was asking earlier.

2) The actual exception is also logged in **run** method of the SparkSQLDriver. If the
logging is set to CONSOLE then the exception is printed on the screen. I hope this is ok ?

I think by default the logging is enabled to console or stdout. If i change the log4j.properties and direct the logging to a file , then we should not see a stack trace.

Michael, do you want this logging completely suppressed for AnalysisException ?